gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineExactDependenciesTest.groovy

```diff
@@ -277,6 +278,26 @@ class BaselineExactDependenciesTest extends AbstractPluginTest {
         with(':checkUnusedConstraints', '--stacktrace', '--write-locks').withDebug(true).build()
     }
 
+    def 'in Gradle >=8.3 you can set the toolchain language version without it being finalised'() {
```

Comment:
this is a failing test in the end?

Reply:
Yeah, I had just stupidly not set the gradle version to 8.4, so it was running with 7.x